extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -117,7 +117,6 @@ public function getWikiVariables() {
 			'homepage' => $this->getHomepageUrl(),
 			'id' => (int) $wgCityId,
 			'language' => [
-				'user' => $wgLang->getCode(),
 				'userDir' => SassUtil::isRTL() ? 'rtl' : 'ltr',
```

Comment:
`userDir` also doesn't make sense - response for this method is cached per wiki.